src/assets/css/header/preheader.css

```diff
@@ -0,0 +1,127 @@
+@font-face {
+  font-family: "SFUI";
+  src: local("SFUI"), url(../../fonts/SFUI.otf) format("opentype");
+}
+
+.panePreHeader {
+  padding: 0px 38px 0px 36px;
```

Comment:
Why not equal padding on left and right?

Reply:
The real nike page had the same padding in `preheader`. That's why I used these padding values.